src/wrap_cl.hpp

```diff
@@ -2574,6 +2574,47 @@ namespace pyopencl
     PYOPENCL_RETURN_NEW_EVENT(evt);
   }
 
+#ifdef CL_DEVICE_P2P_DEVICES_AMD
+  inline
+  event *enqueue_copy_buffer_p2p_amd(
+      platform &plat,
+      command_queue &cq,
+      memory_object_holder &src,
+      memory_object_holder &dst,
+      ptrdiff_t byte_count,
+      py::object py_wait_for)
+  {
+    PYOPENCL_PARSE_WAIT_FOR;
+
+    if (byte_count < 0)
```

Comment:
That's not a great interface IMO. Maybe pass a `py::object` instead that can be `None`, for automatic size determination?

Reply:
I kinda just took what the other buffer transfer functions did, but I guess this is more user facing than the rest of them.